### PR TITLE
Updates to Autoclothing

### PR DESCRIPTION
**Pull request by RosaryMala**
There's a few new features that I want to add, but it works as-is right now. Mainly has issues with over-production.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -72,3 +72,4 @@ tags
 
 # custom plugins
 /plugins/CMakeLists.custom.txt
+out/
```
Comment (myk002) [inline .gitignore:75]: what's this for?

File: library/modules/Buildings.cpp [hunk 2]
```diff
@@ -141,6 +141,8 @@ void buildings_onUpdate(color_ostream &out)
 
     df::job_list_link *link = world->jobs.list.next;
     for (; link; link = link->next) {
+        if (!link->item)
```
Comment (myk002) [inline library/modules/Buildings.cpp:144]: did you just solve a crash here?

**Review by myk002 — COMMENTED** [review]
code looks ok, but there is a distinct lack of changes to `autoclothing` is this the correct branch?

**Comment by RosaryMala** [comment]
You're right, this is the wrong branch, closing this.

**Pull request closed without merging**